feasibility-portal/proxy/.env.default

```diff
@@ -1,3 +1,6 @@
+BACKEND_HOSTNAME=api.datenportal.localhost
+KEYCLOAK_HOSTNAME=auth.datenportal.localhost
+GUI_HOSTNAME=api.datenportal.localhost
```

Comment:
@DeaDurro  GUI_HOSTNAME default should be datenportal.localhost (without the api.)